csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -69,6 +69,47 @@ public Vector3 Centroid
         /// <exception cref="System.ArgumentException">Thrown when coincident vertices are provided.</exception>
         public Polygon(IList<Vector3> vertices) : base(vertices){}
 
+        /// <summary>
+        /// Tests if the supplied 2D point is within the perimeter of this Polygon.
+        /// </summary>
+        /// <param name="point">The 2D Vector3 point to compare.</param>
+        /// <returns>
+        /// Returns true if the supplied Vector3 point is inside this Polygon.
+        /// </returns>
+        public bool Contains(Vector3 point)
+        {
+            var scale = 1024.0;
+            var thisPath = this.ToClipperPath();
+            var intPoint = new IntPoint(point.X * scale, point.Y * scale);
+            if (Clipper.PointInPolygon(intPoint, thisPath) != 1)
+            {
+                return false;
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// Tests if all the supplied 2D Vector3 points fall within the perimeter of this Polygon.
+        /// </summary>
+        /// <param name="points">The collection of 2D Vector3 points to compare.</param>
+        /// <returns>
+        /// Returns true if the supplied Vector3 point is inside this Polygon.
+        /// </returns>
+        public bool Contains(IList<Vector3> points)
```

Comment:
If we're going to keep these methods in here, than they should just call the singular version of the same. Ex:
```
Contains(IList<Vector3> points)
{
  foreach(var p in points) 
  {
    if(!Contains(p))
    {
      return false;
    }
  }
}
```

Reply:
Done.